lms/djangoapps/instructor/views/api.py

```diff
@@ -1267,7 +1267,11 @@ def generate_registration_codes(request, course_id):
     csv_file = StringIO.StringIO()
     csv_writer = csv.writer(csv_file)
     for registration_code in registration_codes:
-        csv_writer.writerow([registration_code.code])
+        full_redeem_code_url = '{base_url}{redeem_code_url}'.format(
+            base_url=site_name,
+            redeem_code_url=reverse('register_code_redemption', kwargs={'registration_code': registration_code.code})
+        )
+        csv_writer.writerow([full_redeem_code_url, registration_code.code])
```

Comment:
@griffresch DO you want the CSV to have the codes in one column, then the URLs with the codes in a second column, or do you just want the URLs at this point?